CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+Thanks for contributing.  A few tips:
+
+* When submitting new files, follow the folder structure listed under **Repository Setup** in [the README](./README.md).
+* Write each new sentence on a new line.
```

Comment:
```suggestion
* Write each new sentence on a new line.
Please don't insert non-sentence or structure-intended extra line breaks. That allows for easier editing and re-arrangement of text parts.
```

Reply:
This suggested text has multiple sentences on a single line :).  I'll take a stab at re-writing.  Also will phrase instructions in the positive tense.

Reply:
Take a look at the latest.

Reply:
> This suggested text has multiple sentences on a single line :). 
That's indeed true, ideal place to actually put multiple sentences on a single line! ;) Thanks for catching it!

> I'll take a stab at re-writing. Also will phrase instructions in the positive tense.
Looks good to me!